cmd/operator/main.go

```diff
@@ -74,14 +80,25 @@ func main() {
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme: scheme,
 		Metrics: metricsserver.Options{
-			// TODO pass metrics to common metrics server
-			BindAddress: "0", // disable metrics service
+			BindAddress: "0", // metrics are served by common metrics server
```

Comment:
0 for a bind address ? empty or 127.0.0.1 no ?

Reply:
https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.19.1/pkg/metrics/server#Options.BindAddress